Cython/Utility/CpdefEnums.pyx

```diff
@@ -48,7 +48,7 @@ if PY_VERSION_HEX >= 0x03040000:
     # create new IntEnum()
     {{name}} = __Pyx_EnumBase('{{name}}', __Pyx_OrderedDict([
         {{for item in items}}
-        ('{{item}}', {{item}}),
+        ('{{item}}', <int>{{item}}),
```

Comment:
Is a plain C `int` correct here? Why not something larger?

Reply:
Think you're right. C can have enums larger than int. There doesn't look too be a single right answer (it can be signed or unsigned) but I'll think if there's a better way to do it

Reply:
I think I've got something better (generated and used the "old" enum conversion function to do this, with the assorted signed checks